src/main/java/io/fabric8/jenkins/openshiftsync/PodTemplateUtils.java

```diff
@@ -159,6 +170,21 @@ public static synchronized void addPodTemplate(PodTemplate podTemplate) {
     }
   }
 
+  protected static PodTemplate setProxyEnvVar(PodTemplate podTemplate) {
+    LOGGER.info("Adding Proxy EnvVar for Template "+podTemplate.getName());
+    List<TemplateEnvVar> envVars = new ArrayList<TemplateEnvVar>();
+    String httpProxy = System.getenv("http_proxy");
```

Comment:
can you set these as constants:
private final static String HTTP_PROXY_ENV_VAR_NAME = "http_proxy";

Reply:
done